script/run_multipart_query

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python
```

Comment:
r? @jklukas for this multipart query pattern

Reply:
```suggestion
#!/usr/bin/env python

"""
Runs multiple queries to temporary tables and then joins those temporary results together.

This is useful for queries that BigQuery deems too complex to run, usually due
to using a large number of subqueries; this pattern allows you to materialize subsets of
columns in multiple different queries so that they stay under the complexity limit, and
then join those results to generate a final wide result.

The query files must be in the same directory and all be prefixed with `part`.
"""
```